_data/projects.yml

```diff
@@ -158,16 +158,11 @@ demo video</a> is available."
       name: "in-toto"
       anchor: "in-toto"
       image: "img/projects/in-toto_diagram.png"
-      status: *starting
+      status: *adopted
       site: "https://in-toto.io"
       description: "Do you know who has handled your software prior to its installation on your machine? Even if developers are careful to secure each step in their products' supply chain, there is little assurance about what happens in-between these steps. The in-toto system holistically enforces the integrity of a software supply chain by gathering and signing information about each step in the process. As such, in-toto provides accountability about how software is written, packaged and distributed...and by who."
-      products: "The in-toto software has already been integrated into several open source
-    projects. Most recently, Arch Linux announced that a patch to check for
-    invalid tags in git will be integrated into the next release of its <a href=\"https://git.archlinux.org/pacman.git/commit/?id=39319c1860d200a9b4a3cc2c6975e3cece502f2d\">pacman</a>
-    utility. We welcome you to download the in-toto
-    instructions, which includes a <a href=\"https://github.com/in-toto/in-toto/tree/develop/demo/\">demo version</a> of our software,
-    or to clone our <a href=\"https://github.com/in-toto/in-toto/\">repository</a> and follow the
-    directions to integrate in-toto into your software project!"
+      products: "The in-toto software has already been integrated into several open
+  source projects. Most recently, Datadog announced the use of <a href=\"https://www.datadoghq.com/blog/engineering/secure-publication-of-datadog-agent-integrations-with-tuf-and-in-toto/\">TUF and in-toto</a> into their agents integration downloader, and a <a href=\"https://salsa.debian.org/reproducible-builds/debian-rebuilder-setup\">constellation of rebuilders</a> are generating in-toto metadata so you can check your Debian packages were built reproducibly when using <a href=\"https://github.com/in-toto/apt-transport-in-toto\">apt</a>. We welcome you to download the in-toto instructions, which includes a <a href=\"https://github.com/in-toto/in-toto/tree/develop/demo/\">demo version</a> of our software, or to clone our <a href=\"https://github.com/in-toto/in-toto/\">repository</a> and follow the directions to integrate in-toto into your software project!"
```

Comment:
A minor language nit-pick from a non-native speaker:  is *"into"* the right preposition when Datadog *"announced the use of ... into ..."*?